jekyll/assets/js/main.js

```diff
@@ -261,10 +258,81 @@ $( document ).ready(function() {
     $("nav.sidebar").toggleClass("open");
   });
 
+  var tooltip = document.querySelector(".server-version-popover");
+
   // Give article headings direct links to anchors
   $("article h2, article h3, article h4, article h5, article h6").filter("[id]").each(function () {
     $(this).append('<a href="#' + $(this).attr("id") + '"><i class="fa fa-link"></i></a>');
   });
+
+  var showEvents = ['mouseover', 'hover', 'mouseenter', 'focus'];
+  var hideEvents = [
+    'mouseout', 
+    'mouseleave', 
+    'blur'
+  ];
+  var clickEvents = ['click'];
+
+  var makePopper = (icon) => Object.assign(icon, {
+    show() {
```

Comment:
Let's add a `trackAction` when the user sees the first tooltip

Reply:
done!